pyomo/core/tests/unit/test_visitor.py

```diff
@@ -167,6 +168,23 @@ def test_identify_mutable_parameters(self):
         self.assertEqual( list(identify_mutable_parameters(
             m.a**m.b[1] + m.b[2]*m.b[3]*m.b[2])), [] )
 
+    def test_identify_mutable_parameters_constants(self):
+        #
+        # SimpleParams and NumericConstants are not recognized
+        #
+        m = ConcreteModel()
+        m.x = Var(initialize=1)
+        m.x.fix()
+        m.p = Param(initialize=2, mutable=False)
+        m.p_m = Param(initialize=3, mutable=True)
+        e1 = (m.x + m.p + 5)
```

Comment:
Can we make `5` be `NumericConstant(5)` (from `pyomo.core.expr.numvalue`)

Reply:
Done. Assumed this was happening under the hood